usaspending_api/disaster/v2/views/count_base.py

```diff
@@ -0,0 +1,60 @@
+from django.db.models import Q
+from usaspending_api.disaster.v2.views.disaster_base import DisasterBase
+
+
+class CountBase(DisasterBase):
+    def is_after_min_date(self):
```

Comment:
`is_after_min_date()` is unnecessary if `self.all_closed_defc_submissions` is used after #2567 is merged

Reply:
bump